.build/db/migrations/20191118083102_create_cla_groups.sql

```diff
@@ -0,0 +1,23 @@
+-- migrate:up
+
+create table cla.cla_groups (
+  id uuid NOT NULL DEFAULT gen_random_uuid(),
+  cla_group_name varchar(255) NOT NULL,
+  foundation_id varchar(255) NOT NULL,
+  created_at BIGINT NOT NULL DEFAULT extract(epoch from now()),
+  updated_at BIGINT NOT NULL DEFAULT extract(epoch from now()),
+  ccla_enabled boolean,
+  icla_enabled boolean,
+  UNIQUE (foundation_id, cla_group_name),
```

Comment:
@dealako  yupp, it's a combination of (project_id, cla_group_name).
So It will make cla_group_name unique in each project but same name can exist in different project.